src/VisualStudio/IntegrationTest/New.IntegrationTests/Options/GlobalOptionsTest.cs

```diff
@@ -64,7 +64,10 @@ public async Task ValidateAllOptions()
             foreach (var language in option.IsPerLanguage ? allLanguages : noLanguages)
             {
                 var key = new OptionKey2(option, language);
-                var currentValue = globalOptions.GetOption<object?>(key);
+
+                // validate that reading the option directly from the persister without falling back to default value works:
+                AssertEx.AreEqual(true, vsSettingsPersister.TryFetch(key, out var currentValue),
+                    message: $"Option '{option.Definition.ConfigName}' failed to load from VS settings.");
```

Comment:
Looks like this already caught something :)

```
Option 'SplitStringLiteralOptions_Enabled' failed to load from VS settings.
Expected:
True
Actual:
False
```

Reply:
Haha, yeah, I think this change is needed first: https://github.com/dotnet/roslyn/pull/66421

Reply:
Actually, seems like by many of the settings are not present in the registry in the default VS image. So we can't assume they are there. This test is not gonna work.